app/views/admin/custom_organisations/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-l">Add Organistions to the Register</h2>
```

Comment:
I think this is misleading - I'd read "Register" as what we're getting from Registers, and we're not adding it back to them.

As I tried to write a simpler sentence, it made me think - this is a pretty convoluted process:
- someone tries to sign up with `Custom Organisation Inc.`
- we get an e-mail
- we add `Custom Organisation Inc.` (and have to spell it exactly right!)
- we go back to the user and tell them to sign up again

I understand why we've ended up with this to start but it makes explaining it in simple terms a pain.

Maybe something like
> ### Allow an organisation access to the admin platform
> Enter the organisation's full name:
> [Text box]
> [Button] Allow organisation